srcpkgs/ivpn/INSTALL.msg

```diff
@@ -0,0 +1,3 @@
+Start ivpn-service:
```

Comment:
Nitpick but perhaps a note indicating that the daemon is mandatory could be useful, i.e., `The ivpn-service must be running in order to use ivpn-cli:`